test/run-container

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+set -eu
+
+IMAGE="${1:-registry.fedoraproject.org/fedora:rawhide}"
+
+podman run --interactive --rm --volume `pwd`:/source --workdir /source "$IMAGE" <<EOF
+set -eux
+
+dnf install -y python3-pip python3-pytest
+
+pip3 install -t /tmp/install .
```

Comment:
Why not just use tox?  This is essentially exactly what it does.

I just pushed an updated version of the config.  In its default invocation, `tox` now uses the site packages for building and installing into a temporary venv and running pytest.

Reply:
Due to this, and I'm a tox n00b :grin: 
```
❱❱❱ tox    
ERROR: pyproject.toml file found.    
To use a PEP 517 build-backend you are required to configure tox to use an isolated_build:    
https://tox.readthedocs.io/en/latest/example/package.html    
```
Thanks for fixing!

Reply:
... or did you?
```
$ tox
tox.exception.ConfigError: ConfigError: 'install_command' must contain '{packages}' substitution
```
This is bog-standard Fedora 38 `python3-tox`.

Reply:
... and if I append `{packages}` to install_command, I get the old message back:
```
ERROR: pyproject.toml file found.
To use a PEP 517 build-backend you are required to configure tox to use an isolated_build:
https://tox.readthedocs.io/en/latest/example/package.html
```

Reply:
That was with tox-3.28.0-1.fc37.noarch. I updated my toolbox to F38 (we just refreshed cockpit/tasks today), using tox-4.4.12-1.fc38.noarch now. It gets a little further:

```
site: install_deps> python3 -m pip install --no-index --no-build-isolation pytest-cov pytest-timeout
ERROR: Could not find a version that satisfies the requirement pytest-cov (from versions: none)
ERROR: No matching distribution found for pytest-cov
```
Hmm.. that worked fine with `pip install`. But when running in a venv (which is absoeffingly the wrong thing to do for this use case..) with e.g. `tox -e py312` it works.

/me grumbles about tox, it's on about minus 3 on my ♥ scale now..